src/sctools/test/test_barcode.py

```diff
@@ -58,3 +62,81 @@ def test_summarize_hamming_distances_gives_reasonable_results(short_barcode_set_
     assert hamming_summary['minimum'] >= 2
     # no barcode can have more hamming distance than length
     assert all(v <= 16 for v in hamming_summary.values())
+
+
+# TEST HashErrorsToCorrectBarcodes
+
+@pytest.fixture(scope='module')
+def trivial_whitelist():
+    barcode_iterable = ['A' * 8]
+    em = barcode.ErrorsToCorrectBarcodesMap._prepare_single_base_error_hash_table(barcode_iterable)
+    return barcode.ErrorsToCorrectBarcodesMap(em)
+
+
+@pytest.fixture(scope='module')
+def truncated_whitelist_from_10x():
+    # note that this whitelist contains 1 non-10x barcode to ensure the presence of a matching
+    # target in the test data.
+    ec = barcode.ErrorsToCorrectBarcodesMap.single_hamming_errors_from_whitelist(
```

Comment:
Can you also replace `ec` with what it stands for? Sorry to nit-pick your variable names!